test/externalTests/zeppelin.sh

```diff
@@ -66,6 +66,10 @@ function zeppelin_test
     sed -i "s|it(\('reverts \)|it.skip(\1|g" math/SignedSafeMath.test.js
     sed -i "s|it(\('reverts \)|it.skip(\1|g" structs/EnumerableSet.behavior.js
     popd
+    # TODO: check
```

Comment:
todo

Reply:
True - although it's a rather safe bet that it's just revert reason heuristics failing.

Reply:
Just had it run again (https://app.circleci.com/pipelines/github/ethereum/solidity/23661/workflows/4202dc2e-60a5-4b55-af8d-c3dce6ac82e8/jobs/1038942) - the error is:
```
      -Transaction reverted and Hardhat couldn't infer the reason. Please report this to help us improve Hardhat.
      +TransparentUpgradeableProxy: admin cannot fallback to proxy target
```

So the test case indeed still reverts, but hardhat doesn't pick up the revert reason anymore - so I think we can safely say this is just testing heuristics failing. I'm actually rather surprised that it's only this one test that has this issue, I'd have expected more...

I'll disable the test again now in any case.